src/main/scala/com/lightbend/kafka/scala/streams/typesafe/util/Materialized.scala

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2018 OpenShine SL <https://www.openshine.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.lightbend.kafka.scala.streams.typesafe.util
+
+import org.apache.kafka.common.serialization.Serde
+import org.apache.kafka.streams.kstream.{Materialized => Base}
```

Comment:
I would rather name it `JMaterialized` or `JavaMaterialized`?
The issue is I had to look at this import to understand what `Base` is.